cms_status_message/models.py

```diff
@@ -0,0 +1,65 @@
+# -*- coding: utf-8 -*-
+# © 2017 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from openerp import models, api
+from openerp.http import request
+from openerp import _
+
+
+class Website(models.Model):
+    _inherit = 'website'
+
+    @property
```

Comment:
You have to fix Travis. I think you can remove the translation test, I was meaning to do a manual check. Translations is a builtin framework system that should be tested in the framework itself. I was just really asking if it worked under `@property` methods, but given it does, it's enough to know it IMHO 😉